test/Npgsql.Tests/TransactionTests.cs

```diff
@@ -338,6 +338,32 @@ public void Savepoint()
             }
         }
 
+        [Test]
+        public async Task SavepointAsync()
+        {
+            using (var conn = OpenConnection())
+            {
+                conn.ExecuteNonQuery("CREATE TEMP TABLE data (name TEXT)");
```

Comment:
Can you give this a more specific name to avoid collisions down the road?

Reply:
Apologies for misunderstanding, but do you mean the temp table or the connection object?

Reply:
Actually, disregard this too. Reviewed this too quickly on my phone and overlooked the `TEMP`. (Erroneously thought you were trying to create a normal table named `data`.)